content/kotlin/concepts/datetime/terms/monthsUntil/monthsUntil.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.monthsUntil()'
+Description: 'Returns the number of months between two instants.'
```

Comment:
Here the instants are two dates right? @theonlygnome if yes then we can use the word `dates` here as it will be more easier to understand it that way.

Reply:
That makes sense.  I've updated the sentence as such.